docs/sphinx/developers/cpp/overview.txt

```diff
@@ -26,57 +26,100 @@ Prerequisites
+| Python Genshi  | 0.7          | 0.6          |    \•    |              |        |
++----------------+--------------+--------------+----------+--------------+--------+
+| Git†           | 2.1.x        | 1.7.x        |    ◦     |              |        |
++----------------+--------------+--------------+----------+--------------+--------+
+| GTest†         | 1.7          | 1.5          |    ◦     |              |        |
++----------------+--------------+--------------+----------+--------------+--------+
+| Doxygen‡       | 1.8          | 1.6          |    ◦     |              |        |
++----------------+--------------+--------------+----------+--------------+--------+
+| Graphviz‡      | 2.x          | 1.8.10       |    ◦     |              |        |
++----------------+--------------+--------------+----------+--------------+--------+
+| Python Sphinx¶ | 1.2.x        | 1.1.x        |    ◦     |              |        |
++----------------+--------------+--------------+----------+--------------+--------+
+| Apache Ant§    | 1.9.x        | 1.8.x        |    ◦     |              |        |
++----------------+--------------+--------------+----------+--------------+--------+
+| Java§          | 1.7.x        | 1.6.x        |    ◦     |              |        |
++----------------+--------------+--------------+----------+--------------+--------+
+| TeX (XeLaTeX)§ | TeXLive 2014 | TeXLive 2012 |    ◦     |              |        |
++----------------+--------------+--------------+----------+--------------+--------+
+
+\•
+  Required for Bio-Formats build; headers may be needed for client
+  build; libraries and any data files required for deployment
+◦
+  Optional for Bio-Formats build; if used for the Bio-Formats build,
+  headers may be required for client build and libraries and any data
+  files required for deployment
 
 \*
-  Optional
+  Optional, needed to build the OpenGL image viewer and client applications
 †
-  Optional, needed to build the API reference
+  Optional, needed only for building
 ‡
-  Optional, needed to build the Sphinx manual
+  Optional, needed to build the API reference
+¶
+  Optional, needed to build the manual pages
+§
+  Optional, needed to build the manual (HTML and PDF)
 
 Due to lacking a package manager, if building on Windows most of the
 above packages will require downloading and installing by hand.  Links
 for these are provided below.
 
+Quick start
+^^^^^^^^^^^
+
+Install the following packages to build Bio-Formats C++.  A subset of
+these packages (or their dependencies) may be used for deployment,
+where the development package headers and tools for building
+documentation etc. are not required.
+
+BSD Ports
+  run ``pkg install devel/boost-all devel/cmake science/hdf5 graphics/png lang/python textproc/py-genshi graphics/tiff textproc/xerces-c3 devel/git devel/googletest math/glm devel/qt5 graphics/graphviz devel/apache-ant java/openjdk7 textproc/py-sphinx print/texlive-full``
```

Comment:
These look great, but I find the "run" to be a bit distracting. If it could be removed or be more visually separated from the rest that'd be great.